packages/grafana-schema/rollup.config.ts

```diff
@@ -34,4 +36,19 @@ export default [
       format: 'es',
     },
   },
+  {
+    input: Object.fromEntries(
```

Comment:
I might be missing something here but I'm not sure we need to introduce another entry here. I think we could change the typescript `declarationDir` to output the types to `"./dist/esm"` then change the `input` above on [L32](https://github.com/grafana/grafana/pull/69678/files#diff-d30a0c133edc9edba399f921e02a203cb5598c2699f43531253d8756a62d5570R32) to `'./dist/esm/index.d.ts'` and we should have the same result.

Even if extra types are available from `dist/esm/index.d.ts` I don't believe they are exposed publicly as the `types` property in package.json will still point to `dist/index.d.ts`.

WDYT @dprokop ?

Reply:
@jackw but wouldn't this result in types conflicts? Panels now export `Options`/`FieldConfig` types that are namespaced only via path.

Reply:
Not sure if I'm following but I'm not suggesting you import from `dist/esm/index.d.ts` just pointing out it would exist in case it's an issue. You'd continue to import in the same way as the scenes PR. e.g.

```
import {
  Options as TimeSeriesOptions,
  FieldConfig as TimeSeriesFieldConfig,
} from '@grafana/schema/dist/esm/raw/composable/timeseries/panelcfg/x/TimeSeriesPanelCfg_types.gen';
import { Options as TableOptions } from '@grafana/schema/dist/esm/raw/composable/table/panelcfg/x/TablePanelCfg_types.gen';
import { Options as BarGaugeOptions } from '@grafana/schema/dist/esm/raw/composable/bargauge/panelcfg/x/BarGaugePanelCfg_types.gen';
```

Reply:
@jackw just pushed an updated version with things we have discussed today:) Thanks for the help again!